openlibrary/catalog/marc/tests/test_data/bin_expect/880_table_of_contents.json

```diff
@@ -0,0 +1,45 @@
+{
+  "publish_date": "2006",
+  "publish_country": "ru",
+  "languages": [
+    "rus"
+  ],
+  "authors": [
+    {
+      "name": "Petrushevskai︠a︡, Li︠u︡dmila",
+      "entity_type": "person",
+      "personal_name": "Petrushevskai︠a︡, Li︠u︡dmila"
+    }
+  ],
+  "other_titles": [
+    "Vremi︠a︡ nochʹ"
+  ],
+  "notes": "Short stories and a novel",
+  "table_of_contents": [
+    {
+      "title": "Rasskazy",
+      "type": "/type/toc_item"
+    },
+    {
+      "title": "Vremi︠a︡ nochʹ : roman",
+      "type": "/type/toc_item"
+    }
+  ],
+  "title": "Zhiznʹ ėto teatr",
+  "subtitle": "[rasskazy, roman]",
```

Comment:
This translates to "[stories, novel]" which isn't really a subtitle, but it seems like the $b subfield is sometimes used this way e.g. [proceedings]

Reply:
@tfmorris Good catch, I hadn't noticed this. I hope I'm not being too heavy handed, but 62ef5ddfe0366ea96994b616b708b88e3ebc5eb5 removes any subtitle that is entirely surrounded in brackets, using an existing regex to handle that general case.